linux/reproducibility/ae_reproducibility_verifier/README.md

```diff
@@ -1,9 +1,9 @@
-# How to Verify Intel(R) Prebuilt AE Reproducibility
+# How to Verify Intel(R) Prebuilt Architectural Enclaves (AE) Reproducibility
 
 ## Purpose
 
-Since Intel(R) prebuilt AEs have been signed by Intel's private key, it's not easy to verify your AE builds against Intel(R) Prebuilt AEs in a common way (e.g. comparing the file checksum).  
-So the doc gives an approach to guarantee Intel(R) prebuilt AEs are:
+Intel(R) prebuilt AEs have been signed by Intel which include the [Enclave Signature Structure](https://download.01.org/intel-sgx/latest/linux-latest/docs/Intel_SGX_Developer_Reference_Linux_2.22_Open_Source.pdf) to the files. Therefore, it's not easy to verify they are reproducible by comparing the checksum of your AE builds with the ones found in Intel(R) prebuilt AEs.  
```

Comment:
1. Better to use below link instead to avoid changing it for each release: 
https://download.01.org/intel-sgx/sgx-linux/2.22/docs/Intel_SGX_Developer_Reference_Linux_2.22_Open_Source.pdf

2. checksum --> checksums

Reply:
make sense